app/views/programs/_form.html.haml

```diff
@@ -27,13 +27,14 @@
               = t[:label]
             - if t[:building]
               = building = sp.association :building, label: 'Building Name', wrapper_html: {class: 'building form-group__indented-01'}, input_html: {class: 'jBuilding'}
+      .form-group.form-group__indented-02
+        = sp.input :weighting_rules_active, label: 'Obey Match-Route Weighting Rules'
```

Comment:
@eanders I added the input here because the partial with the rules in this page was geared towards the program instead of the sub-program. If you think it should be moved into the partial, just let me know and I can make the update.